x-pack/plugins/uptime/common/constants/plugin.ts

```diff
@@ -8,12 +8,17 @@ import { i18n } from '@kbn/i18n';
 
 export const PLUGIN = {
   APP_ROOT_ID: 'react-uptime-root',
-  DESCRIPTION: 'Uptime monitoring',
+  DESCRIPTION: i18n.translate('xpack.uptime.pluginDescription', {
+    defaultMessage: 'Uptime monitoring',
+    description: 'The description text that will be shown to users in Kibana',
```

Comment:
We should change this default description text.

Reply:
This is the text that will be shown in the feature catalogue - I changed the description a bit to reflect that. LMK if you have a recommendation.